node/pkg/watchers/cosmwasm/watcher.go

```diff
@@ -362,9 +368,9 @@ func (e *Watcher) Run(ctx context.Context) error {
 	}
 }
 
-func EventsToMessagePublications(contract string, txHash string, events []gjson.Result, logger *zap.Logger, chainID vaa.ChainID, contractAddressKey string) []*common.MessagePublication {
+func EventsToMessagePublications(contract string, txHash string, events []gjson.Result, logger *zap.Logger, chainID vaa.ChainID, contractAddressKey string, b64Encoded bool) []*common.MessagePublication {
 	// Injective does not base64 encode parameters (as of release v1.11.2).
-	b64Encoded := chainID != vaa.ChainIDInjective
+	// Terra2 no longer base64 encodes parameters.
```

Comment:
Should these comments have moved with the line?

Reply:
Fixed.